src/Middleware/Diagnostics.EntityFrameworkCore/src/Views/DatabaseErrorPage.cshtml

```diff
@@ -94,7 +114,7 @@
                         ErrorApplyingMigrations();
                     };
 
-                    var formBody = "context=@JavaScriptEncode(UrlEncode(Model.ContextType.AssemblyQualifiedName))";
+                    var formBody = "context=" + encodeURIComponent(document.getElementById('applyMigrations').getAttribute('data-assemblyname'));
                     req.open("POST", "@JavaScriptEncode(Model.Options.MigrationsEndPointPath.Value)", true);
```

Comment:
#22517 is about this line not including the request PathBase.

Reply:
Ah yes, I forgot this cuz at one point we were going to remove it all together.